internal/services/recommender.go

```diff
@@ -54,7 +54,13 @@ func ProcessEvent(msg *kafka.Message) {
 		containers := data[0].Kubernetes_objects[0].Containers
 		for _, container := range containers {
 			for _, v := range container.Recommendations.Data {
-				if len(v.Duration_based.Short_term.Notifications) == 0 {
+
+				notifications := v.Duration_based.Short_term.Notifications
+
+				// Below "if" condition is the kruize patch for - https://github.com/kruize/autotune/issues/770
+				// issue - RHIROS-1123
+				// Should be removed once fixed in kruize
+				if !(len(notifications) > 1 && notifications[0].Notificationtype == "info" && notifications[0].Message == "There is not enough data available to generate a recommendation.") {
```

Comment:
```suggestion
				if !(len(notifications) > 1 && notifications[0].Type == "info" && notifications[0].Message == "There is not enough data available to generate a recommendation.") {
```

Looking at `notifications` object from [here](https://github.com/kruize/autotune/issues/770), it says `type` not `notification_type`. 
OR is it something related to golang convention?

Reply:
Yes, it is related to golang - [here](https://github.com/RedHatInsights/ros-ocp-backend/blob/main/internal/types/kruizePayload/common.go#L86)